tests/tests_integration/test_api_client.py

```diff
@@ -430,3 +446,37 @@ def test_delete_empty(self, cognite_client: CogniteClient) -> None:
         res = cognite_client.events.delete(external_id=[])
 
         assert res is None
+
+
+@pytest.fixture()
+def new_asset(cognite_client: CogniteClient) -> Asset:
+    new_asset = AssetWrite(
+        external_id="test_api_client:new_asset:fixture",
+        name="test_asset",
+        description="test_asset",
+        metadata={"test": "test"},
+        source="test",
+    )
+    created = cognite_client.assets.create(new_asset)
+    yield created
+    cognite_client.assets.delete(external_id=new_asset.external_id, ignore_unknown_ids=True)
+
+
+class TestAPIClientUpdate:
```

Comment:
Theses tests are here to ensure updating with the writable classes works as expected.